internal/configurator/configurator.go

```diff
@@ -0,0 +1,96 @@
+/* Package Configurator is responsible for creating tasks from auditjobs mentioned in the config.yaml
+ */
+
+package configurator
+
+import (
+	"errors"
+	"strings"
+	"time"
+
+	"github.com/maplelabs/github-audit/gitprovider"
+	"github.com/maplelabs/github-audit/input"
+	"github.com/maplelabs/github-audit/internal/task"
+	"github.com/maplelabs/github-audit/logger"
+	"github.com/maplelabs/github-audit/utils"
+)
+
+const (
+	// Default Scheduling interval for each audit job.
+	DEFAULTDURATION = time.Duration(5 * time.Minute)
+)
+
+var (
+	log logger.Logger
+	// ErrNoTaskConfigured if there is no audit job configured.
+	ErrNoTaskConfigured = errors.New("no audit task configured")
+)
+
+func init() {
+	log = logger.GetLogger()
```

Comment:
Make sure logs have info on which modue log is coming from. there are logger which privide that.

Reply:
log have the file name from which file it is coming from , should i add for package also?